File: ProjetoCs-Reserva_de_Hotel-main/ReservaDeHotel/Models/Avaliacao.cs
using System.ComponentModel.DataAnnotations;

namespace ReservaDeHotel.Models;
public class Avaliacao
{
    [Key]
    public int IdAvaliacao {get;set;}
    public int IdHotel { get; set;} // aqui teria que usar uma ForeignKey
    public int AvaliacaoEstrelas { get; set;}
    public string? Comentario { get; set;}
    public DateTime DataAvaliacao { get; set;}
}

File: ReservaHotelFront/src/app/Avaliacao.ts
import { Hotel } from './Hotel';

export class Avaliacao {
    IdAvaliacao: number = 0;
    AvaliacaoEstrelas: number = 0;
    Comentario: string | null = null;
    DataAvaliacao: Date = new Date();
    IdHotel: number = 0;
    Hotel: Hotel | null = null;
}
